spec/system/admin/config/admin_updates_smtp_settings_spec.rb

```diff
@@ -0,0 +1,111 @@
+  context "when Sendgrid is not enabled and SMTP is not enabled" do
+    before do
+      allow(ForemInstance).to receive(:sendgrid_enabled?).and_return(false)
+      allow(Settings::SMTP).to receive(:address).and_return(nil)
+      allow(Settings::SMTP).to receive(:user_name).and_return(nil)
+      allow(Settings::SMTP).to receive(:password).and_return(nil)
+      visit admin_config_path
+    end
+
+    it "does not show the 'Use my own email server' checkbox" do
+      within("form[data-testid='emailServerSettings']") do
+        expect(page).not_to have_content("Use my own email server")
+      end
+    end
+
+    it "shows the SMTP Form" do
+      within("form[data-testid='emailServerSettings']") do
+        expect(page).to have_selector(".js-custom-smtp-section")
+        expect(page).not_to have_selector(".js-custom-smtp-section.hidden")
+      end
+    end
+  end
+
+  context "when Sendgrid is enabled and SMTP is not enabled" do
+    before do
+      allow(ForemInstance).to receive(:sendgrid_enabled?).and_return(true)
+      allow(ForemInstance).to receive(:email).and_return("yo@forem.com")
+      allow(Settings::SMTP).to receive(:address).and_return(nil)
+      allow(Settings::SMTP).to receive(:user_name).and_return(nil)
+      allow(Settings::SMTP).to receive(:password).and_return(nil)
+      visit admin_config_path
+    end
+
+    it "shows the checkbox to allow one to toggle ones own server" do
+      within("form[data-testid='emailServerSettings']") do
+        expect(page).to have_content("Use my own email server")
+      end
+    end
+
+    it "shows a description" do
+      within("form[data-testid='emailServerSettings']") do
+        # rubocop:disable Layout/LineLength
+        expect(page).to have_content("As a Forem Cloud client, we provide an email server managed by the Forem team. All settings are managed by us and the from and reply email addresses are set as yo@forem.com. However, you can override this to use your own email server.")
+        # rubocop:enable Layout/LineLength
+      end
+    end
+
+    it "does not show an SMTP Form" do
+      within("form[data-testid='emailServerSettings']") do
+        expect(page).to have_selector(".js-custom-smtp-section.hidden")
+      end
+    end
+  end
+
+  context "when Sendgrid is not enabled and SMTP is enabled" do
+    before do
+      allow(ForemInstance).to receive(:sendgrid_enabled?).and_return(false)
+      allow(Settings::SMTP).to receive(:address).and_return("smtp.gmail.com")
+      allow(Settings::SMTP).to receive(:user_name).and_return("jane_doe")
+      allow(Settings::SMTP).to receive(:password).and_return("abc123456")
+      visit admin_config_path
+    end
+
+    it "does not show the 'Use my own email server' checkbox" do
+      within("form[data-testid='emailServerSettings']") do
+        expect(page).not_to have_content("Use my own email server")
+      end
+    end
+
+    it "shows the SMTP Form" do
```

Comment:
Quick, non-blocking suggestion: same as above comment!
```suggestion
    it "shows the SMTP Form", :aggregate_failures do
```